packages/aragon-api/src/index.js

```diff
@@ -409,6 +409,7 @@ export class AppProxy {
 
     const getCurrentEvents = (fromBlock) => merge(
       this.events(fromBlock),
+      this.getForwardedActions(),
```

Comment:
@Quazia I've implemented the forwarded actions subscription in the store here This way we won't be storing any forwarded actions state inside the app's state cache. All forwardedAction state will only persist within its own wrapper cache.